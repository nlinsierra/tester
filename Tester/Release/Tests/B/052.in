5H 5C 8H JC TH 4S TD 6S QD 3D KD TS JD AH 4H 6H AS 9H 8C JS 2S KS AD 9D QH KH QS QC 9C 7D AC 7C 4D KC 3S 6C 8S 7H 5S 3H 2D 2C 8D JH 4C 7S 6D TC 9S 2H
