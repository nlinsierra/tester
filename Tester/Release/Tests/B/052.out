2
3C 5D
